source/unified-test-format/unified-test-format.md

```diff
@@ -376,6 +376,12 @@ The structure of this object is as follows:
 
     If false, tests MUST NOT run if CSFLE is supported. If this field is omitted, there is no CSFLE requirement.
 
+    The structure of the `csfle` object is as follows:
+
+    - `minLibmongocryptVersion`: Optional string. The minimum libmongocrypt (inclusive) required to successfully run the
+        tests. If this field is omitted, there is no lower bound on the required libmongocrypt version. The format of this
+        string is defined in [Version String](#version-string).
```

Comment:
This seems fine.

[`runOnRequirement-csfle-type.yml`](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/tests/invalid/runOnRequirement-csfle-type.yml) remains as-is, because you're making the type of `csfle` _more_ permissive, but I think you can add a new test for your restriction on the object value's structure.